tests/platform_tests/reboot_timing_constants.py

```diff
@@ -30,7 +30,9 @@
     "COMMON": {
         "PORT_INIT|Start": re.compile(r'.*NOTICE swss#orchagent.*initPort: Initialized port.*'),
         "PORT_READY|Start": re.compile
```

Comment:
Is `PORT_READY|Start` still used or we only depend on `Start-changes-only`

Reply:
Yes, see below. The `offset_from_kexec` and `time_span` dictionaries in the post-reboot analysis is hardcoded to use the `Start` event, and I want it to still match `up to up`.